spinalcordtoolbox/reports/qc.py

```diff
@@ -618,6 +622,7 @@ def generate_qc(fname_in1, fname_in2=None, fname_seg=None, angle_line=None, args
     each slice, for slice that don't have an angle to display, a nan is expected. To be used for assessing cord orientation.
     :param args: args from parent function
     :param path_qc: str: Path to save QC report
+    :param fps: float: Frame rate for output gif images
```

Comment:
```suggestion
    :param fps: float: Frame per second for output gif images
```

Reply:
The only reason I put it in float is because I tought maybe someone might want a slow frame rate like 0.5, also the function imageio.mimsave() to save gifs takes the fps input as float.

Reply:
ah yes! i makes perfect sense 👌